src/cli/default-error-handler.ts

```diff
@@ -251,9 +251,10 @@ once your changes are merged with the new remote version, you can tag and export
       }
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part